code/game/machinery/doors/airlock.dm

```diff
@@ -637,13 +637,10 @@ About the new airlock wires panel:
 		return 0
 	if(hasShocked)
 		return 0	//Already shocked someone recently?
-	if(..())
-		hasShocked = 1
-		sleep(10)
+	. = ..()
+	hasShocked = 1
+	spawn(10)
 		hasShocked = 0

```

Comment:
Should this not default to return `TRUE`?

Reply:
It will return whether or not the door actually shocked someone. I tried using setClickCooldown, but that didn't appear to be processing correctly (I set it to user.get_click_cooldown() [sic], and could spam click it for lots of sparks when it should only have been processing one click every 0.8 seconds, similar behaviour for giving it 10 directly)